pytest.ini

```diff
@@ -1,6 +1,6 @@
 [pytest]
 norecursedirs=dist build .tox .eggs
-addopts=--doctest-modules --flake8 --cov
+addopts=--doctest-modules --cov
```

Comment:
I suspect you're removing pytest-flake8 because a recent release of flake8 broke the plugin. See jaraco/skeleton for how I've addressed the issue (https://github.com/jaraco/skeleton/commit/c64902b8cafa8062398ef173278a21b042b03a77).

Reply:
That issue with flake8 breaking it motivated me to drop flake8, however I was planning to drop flake8 eventually anyway because black is more consistent and popular. I also integrated black into GitHub so that any commits that don't follow black style automatically get changed to black style on merge/push.